datasets/get_test_data.sh

```diff
@@ -44,19 +39,41 @@ https://s3.us-east-2.amazonaws.com/rapidsai-data/cugraph/benchmark/hibench/hiben
 benchmark
 "
 
+BENCHMARK_DATASET_DATA="
+# ~90s download - these are used for benchmarks runs (code in <cugraph root>/benchmarks)
+https://rapidsai-data.s3.us-east-2.amazonaws.com/cugraph/benchmark/benchmark_csv_data.tgz
```

Comment:
Do we keep the list of what's in this archive? 
We will soon add benchmark datasets, not sure we will rebuild the whole archive for this (we could just add new links). It would be great to keep track of what's in there.

Reply:
We could think of something similar to this section for python testing datasets https://github.com/rapidsai/cugraph/tree/branch-0.15/datasets#python

Reply:
@afender I agree since I think there's probably some additional cleanup we could do to all the datasets to reorg them so they make better sense, have less overlap, etc. For instance, right now there's a folder created for testing datasets called "benchmarks", which is confusing.
I'll take a pass at updating the doc you linked as a start.